js/constants.js

```diff
@@ -256,3 +256,8 @@ export const MAP_COORDINATES = {
   LNG: 78.476681027237,
   LAT: 22.1991660760527,
 };
+export const BASE_URL = 'https://api.realdevsquad.com/';
+export const JOIN_POST_URL = BASE_URL + 'users/self/intro';
```

Comment:
Nit: we can use ES6 template literals 
```suggestion
export const JOIN_POST_URL = `${BASE_URL}/users/self/intro`;
```